tests/baselines/reference/smartSelection_JSDocTags10.baseline

```diff
@@ -0,0 +1,29 @@
+/**
+ * @template T
+ * @extends {/**/Set<T>}
+ */
+class A extends B {
+}
+
+
+             Set    
+
+
+             Set<T> 
```

Comment:
It’s interesting that this one doesn’t include a step with just `{Set<T>}` like pretty much all the others do. Was there a rationale there or is it a coincidence? FWIW, I prefer this sequence—skipping `{Set<T>}`—because for one of these tags that takes a type, you always want the curly braces. Suppose that in

```js
/**
 * @returns {string}
 */
```

you selected `{string}` and were going to type something else—I think everything you could want to type would still start with `{`, so I think it would be best to jump from `string` to `@returns {string}`.

Reply:
@andrewbranch I think it is a coincidence. I'll check the option jumping from `string` to `@returns {string}`.